rust/ql/src/queries/unusedentities/UnusedValue.ql

```diff
@@ -21,4 +21,4 @@ where
   // avoid overlap with the unused variable query
   not isUnused(v) and
   not v instanceof DiscardVariable
-select write, "Variable is assigned a value that is never used."
+select write, "Variable '" + v + "' is assigned a value that is never used."
```

Comment:
Thanks, I've been meaning to make this change!

Reply:
We could even make this is link to the variable right? Something like
```suggestion
select write, "Variable '$@' is assigned a value that is never used.", v, v.toString()
```

Reply:
Yep, that will make it clickable as well. :+1:

Reply:
I'll add it.